pages/roles-and-teams.md

```diff
@@ -2,60 +2,21 @@
 title: Roles and teams
 permalink: /roles-and-teams/
 ---
-Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles on a number of different teams across 18F. Below are a few of the roles that Innovation Specialists may fill at 18F. This is not intended to be a comprehensive list but for those that are interested in joining the team, hopefully something on this list excites you!
+Although everyone is officially hired into the GSA as an Innovation Specialist, our Innovation Specialists fill a wide range of roles across 18F, from software engineers and product leads to UX and visual designers. Below is a list of our current needs but they change frequently, so check back often!
 
-### Comms Team
-
-**Director of Outreach**
-
-The Director of Outreach drives a comprehensive internal and external communications strategy for 18F. This includes editorial and content on our website, social media, events/conferences, messaging, and general community evangelism.
+### Communications Team
+[**Director of Outreach**](https://pages.18f.gov/joining-18f/roles-and-teams/positions/director-of-outreach/)
 
 ### Consulting Team
+**Design and Product Strategist** *(Role profile coming soon)*
```

Comment:
"Role profile" is jargony and inconsistent with the way that the Director of Outreach position is listed. Also, this approach is overriding the site's means of dynamically displaying open positions. Merging for now and will make a follow-up ticket.